src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetCopilotHelper.java

```diff
@@ -292,10 +293,19 @@ else if (event.getKeyCode() == KeyCodes.KEY_BACKSPACE)
                      {
                         display_.removeGhostText();
                      }
+                     else if (event.getKeyCode() == KeyCodes.KEY_RIGHT &&
+                              (event.getCtrlKey() || event.getMetaKey()))
```

Comment:
Is this too aggressive? Should I prefer Meta only on macOS, and Ctrl only on other OSes?